.devcontainer/devcontainer.env

```diff
@@ -0,0 +1,17 @@
+#Database setup
+MYSQL_HOST=127.0.0.1
+MYSQL_DATABASE=friendica
+MYSQL_USER=friendica
+MYSQL_PASSWORD=friendica
+
+#Webserver setup
+ServerName=192.168.56.10.xip.io
```

Comment:
What's this domain name?

Reply:
I think it is the same, that the Vagrant dev VM ends with having for local access - @fabrixxm had introduced it IIRC

Reply:
Exactly. This I found in the existing Vagrant setup and I wanted to change as little as possible when creating the dev container setup. It does not have any impact though as the connection to the dev container's httpd is done by port forwarding to localhost:8080.